alfresco-java-rest-api/alfresco-java-rest-api-lib/pom.xml

```diff
@@ -43,6 +43,7 @@
     <module>generated/alfresco-governance-classification-rest-api</module>
     <module>generated/alfresco-search-rest-api</module>
     <module>generated/alfresco-search-sql-rest-api</module>
+    <module>alfresco-event-gateway-api</module>
```

Comment:
Is there any reason for this project not being inside the "generated" folder?

Reply:
Yeap, see here: https://github.com/Alfresco/alfresco-java-sdk/pull/68/files#diff-d731c1d933ec9b1e341b0f289bcb520fc77c5543509bb88553836e308e81f0b6R33